test/test_common.rb

```diff
@@ -29,4 +37,16 @@ def test_underscore_entity
       assert_equal(Kubeclient::ClientMixin.underscore_entity(singular), plural)
     end
   end
+
+  def test_format_datetime_with_string
+    value = '2018-04-27T18:30:17.480321984Z'
+    formatted = client.send(:format_datetime, value)
+    assert_equal(formatted, value)
+  end
+
+  def test_format_datetime_with_time
```

Comment:
last thing: can you also test with a Time instance

Reply:
Done